pkgs/top-level/all-packages.nix

```diff
@@ -26204,6 +26204,10 @@ with pkgs;
 
   fwts = callPackage ../os-specific/linux/fwts { };
 
+  gasket = callPackage ../os-specific/linux/gasket {
+    inherit (linuxPackages) kernel;
+  };
```

Comment:
Shouldn't this then go to os-specific packages?

Reply:
I wasn't 100% sure where I was supposed to put this.  Is this supposed to go into a different file like `linux-kernels.nix`?

Reply:
Oh, wait, it is actually there already.